### ISSUE TITLE
I'm using anaconda and every thing is a mass..

### ISSUE THREAD

**Issue by menglin0320** [zplizzi/tensorflow-fast-rcnn#3]
I'm trying to follow the instruction [here](https://www.tensorflow.org/versions/r0.10/how_tos/adding_an_op/index.html#with-tensorflow-binary-installation)
But I don't know where should I put the build_user_op.sh and call it if I'm using anaconda, 
I found /home/menglin/anaconda2/envs/tensorflow_env/lib/python2.7/site-packages/tensorflow/include 
is a pretty good choice
and I put your user_ops on 
/home/menglin/anaconda2/envs/tensorflow_env/lib/python2.7/site-packages/tensorflow/include/ tensorflow/core/
But that doesn't work. And I don't know what to do now

**Comment by zplizzi** [comment]
I've not used anaconda, but I think it should be the same.

Clone this repository into any folder on your computer, and don't move anything. Simply run the build_user_op.sh file, and it should compile the ops. 

Then, you just have to import these compiled files from your code. Take a look at my example [here](https://github.com/zplizzi/tensorflow-fast-rcnn/blob/master/tensorflow/examples/roi_pooling/demo.ipynb), in the second code block - see how I'm importing the `.so` files? Do the same thing, but provide the path to wherever on your system the compiled files are located.

**Comment by menglin0320** [comment]
---

NotFoundError                             Traceback (most recent call last)
<ipython-input-2-ba2dd6353e67> in <module>()
      5 # Import the forward op
      6 roi_pooling_module = tf.load_op_library(
----> 7     home + "/anaconda2/envs/tensorflow_env/lib/python2.7/site-packages/tensorflow/include/tensorflow/core/user_ops/roi_pooling_op.so")
      8 roi_pooling_op = roi_pooling_module.roi_pooling
      9 

/home/menglin/anaconda2/envs/tensorflow/lib/python2.7/site-packages/tensorflow/python/framework/load_library.pyc in load_op_library(library_filename)
     69           return _OP_LIBRARY_MAP[library_filename]
     70       # pylint: disable=protected-access
---> 71       raise errors._make_specific_exception(None, None, error_msg, error_code)
     72       # pylint: enable=protected-access
     73   finally:

NotFoundError: libcudart.so.8.0: cannot open shared object file: No such file or directory

I got this error

**Comment by menglin0320** [comment]
The problem is on the step running these two cammands
bazel build -c opt --config=cuda //tensorflow/core/user_ops:roi_pooling_op.so  
bazel build -c opt --config=cuda //tensorflow/core/user_ops:roi_pooling_op_grad.so  
in build_user_op.sh, I feel I should do it on the root of the tensorflow I'm using instead of do it on the Cloned repository. But I don't know where is the root since the lay out of tensorflow in anaconda is pretty different from normal tensorflow.
As I said I found the include folder highly possible to correspond to the root file on the cloned repository, but that doesn't work. I copied the script to the include file and tried to run it. But I got 
The 'build' command is only supported from within a workspace.
The 'build' command is only supported from within a workspace.
So I'm very confused now
